pkg/generic/thrift/http_amd64.go

```diff
@@ -0,0 +1,68 @@
+//go:build amd64 && go1.16
```

Comment:
You'd better rename the file name to http_go116_amd64.go if it only supports go1.16